packages/synapse-react-client/src/components/widgets/query-filter/EnumFacetFilter/EnumFacetFilterDropdown.tsx

```diff
@@ -15,20 +22,28 @@ function EnumFacetFilterSelectBox(
 ) {
   const { menuText, children } = props
 
-  const [isShowDropdown, setIsShowDropdown] = useState<boolean>(false)
-  const onToggle = () => setIsShowDropdown(!isShowDropdown)
-
   return (
-    <Dropdown
+    <Select
       className={'EnumFacetFilter EnumFacetFilterSelect'}
-      show={isShowDropdown}
-      onToggle={onToggle}
+      value={menuText}
+      renderValue={() => menuText}
+      onChange={() => {
+        // noop - the children will handle change events
+      }}
     >
-      <Dropdown.Toggle className="secondary-caret" variant="gray-select">
-        {menuText}
-      </Dropdown.Toggle>
-      <Dropdown.Menu>{children}</Dropdown.Menu>
-    </Dropdown>
+      {/* Hack to prevent Select warnings - add a hidden MenuItem */}
+      <MenuItem value={menuText} sx={{ display: 'none' }} />
+      <Box
+        sx={{
+          // hack to expand the dropdown menu width for this component only
```

Comment:
This is not really a hack, compared to the above :) 
```suggestion
          // expand the dropdown menu width for this component only
```